numpy/ma/tests/test_core.py

```diff
@@ -1990,6 +1991,177 @@ def test_datafriendly_mul_arrays(self):
         assert_equal(a, [[1, 1], [3, 3]])
         assert_equal(a.mask, [[0, 1], [0, 1]])
 
+    def test_inplace_addition_scalar_uint8(self):
```

Comment:
Would be better to loop through a set of relevant types. Could also loop through the operations, althought the asserts differ for those. See the [operator module](https://docs.python.org/2/library/operator.html).